### ISSUE TITLE
Vorschlag: Exaktifizierung des Errors

### ISSUE THREAD

**Issue by lenaflz** [kovzol/aris#21]
Wenn die eingegebene Aussage falsch ist, wird nach Drücken des Dreieck-Buttons der Rahmen in rot oder grün gefärbt. Im unteren Teil des Bildschirms erscheint zugleich der Hinweis: "Error in line 1 - The sentence has syntactical errors."
Hier würden exakte Beschreibungen von Errors wie "No connection line is selected." den Umgang mit der Software erleichtern.

**Comment by kovzol** [comment]
Indeed, the error messages are not straightforward. It would be great to get some more explanations.
